.circleci/workspace_status.sh

```diff
@@ -25,3 +28,15 @@ echo "GIT_SHA ${git_sha}"
 # Tag name, or GIT_SHA if not on a tag
 git_ref=$(git describe --tags --no-match --always --abbrev=40 --dirty | sed -E 's/^.*-g([0-9a-f]{40}-?.*)$/\1/')
 echo "GIT_REF ${git_ref}"
```

Comment:
`GIT_*` variable seem like they are no longer necessary. Doesn't hurt to keep them for potential future use, but up to you.

Reply:
Agree, but see next comment.

Should we follow through with Bazel docker, we could use them again.